src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java

```diff
@@ -0,0 +1,37 @@
+package analyzer.exercises.loglevels;
+
+import analyzer.Comment;
+
+import java.util.Map;
+
+/**
+ * @see <a href="https://github.com/exercism/website-copy/blob/main/analyzer-comments/java/log-levels/use_substring_method.md">Markdown Template</a>
+ */
+class UseSubstringMethod extends Comment {
+    private final String callingMethod;
+    private final String methodToCall;
```

Comment:
Isn't this always `substring`? I don't think the Markdown template needs a variable for that.

Reply:
Done!